internal/data/hosts/hosts.go

```diff
@@ -49,6 +56,38 @@ func (r *hostsRepo) Save(ctx context.Context, model *biz.Host) (*biz.Host, error
 			return nil, err
 		}
 	}
+
+	if r.Authz != nil {
+		rels := []*kessel.Relationship{{
+			Resource: &kessel.ObjectReference{
+				Type: &kessel.ObjectType{
+					Name:      "integration",
+					Namespace: "notifications",
+				},
+				// convert the ID to a string
+				Id: strconv.FormatInt(model.ID, 10),
+			},
+			Relation: "workspace",
+			Subject: &kessel.SubjectReference{
+				Subject: &kessel.ObjectReference{
+					Type: &kessel.ObjectType{
+						Name:      "workspace",
+						Namespace: "rbac",
+					},
+					Id: model.Metadata.Workspace,
```

Comment:
Do we need to check if the caller has access to the workspace? or this check is assumed in this call?

Reply:
There should be checks that the reporter is allowed to report for this resource type. That won't be workspace specific.

Workspace would be relevant for the end user. End use access checks have come up a few times in the last few weeks. I have a proposal that we should but we can leave it off of this PR. I'll share that with you tomorrow.

Reply:
for now, we have agreed to not do any checks